apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -160,6 +160,11 @@
     "module": {
       "description": "Module that this assessment belongs to, as defined in infoCourse.json.",
       "type": "string"
+    },
+    "sharedPublicly": {
```

Comment:
we are going to use `sharedSourcePublicly` for this instead

Reply:
and shared_source_publicly in the database (similar to what is done for questions here: https://github.com/PrairieLearn/PrairieLearn/pull/10029)

Reply:
Do we want to replace all instances of sharedPublicly  on assessments with sharedSourcePublicly?

Reply:
Like throughout the whole branch?

Reply:
I changed the name throughout the whole branch and changed the migration accordingly

Reply:
Yes, that's great thanks